.github/workflows/test-nightly.yml

```diff
@@ -15,7 +15,7 @@ on:
 
 jobs:
   test-state-sync-mocha:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-8
```

Comment:
[question] goreleaser also runs on ubuntu-8. Should we be concerned about resource contention and/or GoReleaser running slower b/c this test will be using the instance?

Ref: https://github.com/rootulp/celestia-app/blob/e3861acd63e153eb7f74cec7a443c2624391ab1a/.github/workflows/goreleaser.yml#L11